### PR TITLE
Prime 1: Magma Pool fixes

### PR DESCRIPTION
**Pull request by EthanArmbrust**
-Added requirement for heat runs trick or heat resisting suit to all connections in the room  
-Added standable terrain method for crossing the room https://www.youtube.com/watch?v=eklXYB2qJ9E
-Removed gravity suit and lava damage requirement from Door to Meditation Fountain -> Door to Training Chamber Access, as the standable terrain method in the bullet point above has fewer requirements at the same difficulty levels

### CODE REVIEW

File: randovania/data/json_data/prime1/Chozo Ruins.txt [hunk 1]
```diff
@@ -540,28 +540,46 @@ Asset id: 1226570426
...
   > Door to Training Chamber Access
       Any of the following:
           Grapple Beam and Heat-Resisting Suit
           All of the following:
-              Space Jump Boots and Heated Rooms Damage ≥ 99
+              Space Jump Boots
               Any of the following:
-                  Gravity Suit and L-Jump (Intermediate) and Movement (Intermediate) and Standable Terrain (Intermediate) and Lava Damage ≥ 50
+                  All of the following:
+                      # https://www.youtube.com/watch?v=eklXYB2qJ9E
```
Comment (toasterparty) [inline randovania/data/json_data/prime1/Chozo Ruins.txt:570]: this video shows the wrong direction

Comment (toasterparty) [reply randovania/data/json_data/prime1/Chozo Ruins.txt:570]: Also, I would argue that the gravity suit L-Jump is easier in trick requirements, and so is worth modeling the lava damage for.

(L-Jump intermediate, Standable beginner)

Comment (EthanArmbrust) [reply randovania/data/json_data/prime1/Chozo Ruins.txt]: I've addressed both of these comments in the latest commit

**Comment by toasterparty** [comment]
@EthanArmbrust please resolve conflicts. Thanks!

**Review by toasterparty — APPROVED** [review]

**Pull request merged**